crates/tuono/src/scaffold_project.rs

```diff
@@ -160,6 +159,31 @@ pub fn create_new_project(folder_name: Option<String>, template: Option<String>)
     outro(folder);
 }
 
+fn generate_tree_url(
+    select_main_branch_template: Option<bool>,
+    client: &Client,
+    cli_version: &str,
+) -> String {
+    if select_main_branch_template.unwrap_or(false) {
+        format!("{}main?recursive=1", GITHUB_TUONO_TAG_COMMIT_TREES_URL).to_string()
```

Comment:
Does this need to be returned as string?

```suggestion
        format!("{}main?recursive=1", GITHUB_TUONO_TAG_COMMIT_TREES_URL)
```